tests/smoke.rs

```diff
@@ -82,6 +82,19 @@ fn maildir_list() {
     })
 }
 
+#[test]
+fn maildir_list_subdirs() {
+    with_maildir("maildir2", |maildir| {
+        let mut iter = maildir.list_subdirs();
+        let first = iter.next().unwrap().unwrap();
```

Comment:
This test is relying on the iterator returning subdirs in a specific order (contrary to the docs on the method). It would be better to collect the iterator into a vec and then check that it contains exactly the two expected entries.

Reply:
:heavy_check_mark: